odoo/sql_db.py

```diff
@@ -245,9 +256,22 @@ def execute(self, query, params=None, log_exceptions=None):
             encoding = psycopg2.extensions.encodings[self.connection.encoding]
             _logger.debug("query: %s", self._obj.mogrify(query, params).decode(encoding, 'replace'))
         now = time.time()
+        catch_traceback_query = bool(os.environ.get('catch_traceback_query', ''))
         try:
             params = params or None
             res = self._obj.execute(query, params)
+            if catch_traceback_query:
+                for query_to_catch in QUERY_TO_CATCHS:
+                    query = self._obj.query.replace(b"\n", b" ").replace(b"\'", b'"')
+                    if query_to_catch.lower() in query.lower():
+                        tcbk = traceback.extract_stack()
+                        key = hashlib.md5(repr(tcbk).encode('UTF-8') + query_to_catch).hexdigest()
+                        TRACEBACK_GROUP[key]['count'] += 1
+                        TRACEBACK_GROUP[key]['tcbk'] = tcbk
+                        tcbk_str = ""
+                        for key, tcbk in sorted(TRACEBACK_GROUP.items(), key=lambda x: x[1]['count'], reverse=True):
+                            tcbk_str += "%d,%s,%s\n" % (tcbk['count'], '\n,'.join(map(lambda a: repr(a), tcbk['tcbk'])), query_to_catch)
+                        _logger.info("query executed: %s" % tcbk_str, exc_info=True)
```

Comment:
I think you should addd `self._obj.query` in the logger in order to know what was the real query executed

Use lazzy logger

```suggestion
                        _logger.info("query executed: %s", tcbk_str, exc_info=True)
```